generate-commitish.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#Generates a commitish string for climate explorer and stores it in 
+# the CE_CURRENT_VERSION environment variable.
+
+VERSIONTAG="$(git describe --tags --abbrev=0)"
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+COMMITSHA="$(git log -1 --format=%h)"
+
+export CE_CURRENT_VERSION="Climate Explorer $VERSIONTAG ($BRANCH: $COMMITSHA)"
```

Comment:
Drop the export here statement here, and drop the "Climate Explorer" prefix from the version information.
Just output the info we need straight-up:

```bash
echo  $VERSIONTAG ($BRANCH: $COMMITSHA)
```